plugins/nf-amazon/src/main/nextflow/cloud/aws/nio/S3FileSystemProvider.java

```diff
@@ -535,7 +535,7 @@ public void copy(Path source, Path target, CopyOption... options)
 		 * "copying directories is not yet supported: %s", target); // TODO
 		 */
 		ImmutableSet<CopyOption> actualOptions = ImmutableSet.copyOf(options);
-		verifySupportedOptions(EnumSet.of(StandardCopyOption.REPLACE_EXISTING),
+		verifySupportedOptions(EnumSet.of(StandardCopyOption.COPY_ATTRIBUTES, StandardCopyOption.REPLACE_EXISTING),
```

Comment:
This change doesn't actually enable copy attributes for S3, just prevents a failure. We could remove it, I would just add a warning to the docs that copy attributes only works for loca/shared filesystem

Reply:
Agree on remove it and add a warning that's only honoured by local/shared file systems